src/features/technician-details/dashboard/components/work-order-card.tsx

```diff
@@ -140,11 +158,67 @@ export default function WorkOrderCard({ workOrder }: WorkOrderCardProps) {
           </div>
         </div>
       </div>
+
+      <div className="col-span-6">
+        <div className="flex flex-col">
+          <SmallLabel>Client Address</SmallLabel>
+          <span className="text-xs font-semibold md:text-sm">
+            {workOrder.client.address_line1}, {workOrder.client.address_line2},{" "}
+            {workOrder.client.city}, {workOrder.client.state}{" "}
+            {workOrder.client.postal_code}
+          </span>
+        </div>
+      </div>
+
       {isExpanded && (
```

Comment:
Is there a benefit to having it here vs splitting the file and having them shorter? My intention was to make the files smaller